scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,24 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
+  while read -r line; do
+    comment=$(echo "$line" | cut -c1)
+    if [ ! "$comment" = "#" ]; then               # lines starting with # get ignored
+      owner=$(echo "$line" | awk '{ print $1 }')
+      perms=$(echo "$line" | awk '{ print $2 }')
+      file=$(echo  "$line" | awk '{ print $3 }')
+      mkdir -p ../rootfs"$(dirname "$file")"
```

Comment:
why the `..`?
and please add the `/rootfs` inside the quoted string, like so: `"/rootfs/$(dirname "${file}")"`